README.md

```diff
@@ -208,6 +208,32 @@ const getMarkdown = () => {
 - *Ruby* - [`<<~` Heredocs](https://www.rubyguides.com/2018/11/ruby-heredoc/)
 - *Swift* - [multiline string literals](https://github.com/mmkal/proposal-multi-backtick-templates) using triple-quotes - strips margin based on whitespace before closing delimiter
 
+## Syntax Alternatives Considered
+
+Some potential alternatives to the multi-backtick syntax:
+
+- A built-in runtime method along the lines of the `dedent` library, e.g. `String.dedent`:
+  - Pros:
+    - Easier to implement
+    - No new syntax
+  - Cons:
```

Comment:
The biggest con is whether the [isTemplateObject](https://github.com/tc39/proposal-array-is-template-object) would propagate to the output. This is because both the original template object and the dedented object are accessible:

```js
function extract(tag) {
  return tag`foo`;
}

const original = extract(strings => strings);
const dedented = extract(String.dedent(strings => strings));
```

For whitespace-sensitive languages, this _could_ be a security vulnerability.

However with a syntax form, only the dedented template object would be available at runtime.